samples/maps/demo/flowmap-ship-route/demo.html

```diff
@@ -0,0 +1,8 @@
+<script src="https://code.highcharts.com/maps/highmaps.js"></script>
+<script src="https://code.highcharts.com/maps/modules/exporting.js"></script>
```

Comment:
Same as the previous one, it doesn't affect the code at all - but it might improve the readability, when we "sort" the modules, as maps related, exporting related, accessibility.

But I'd like to hear others opinion on this one.